MAINTAINERS.md

```diff
@@ -19,3 +19,7 @@ The maintainers [watch](https://github.com/Particular/NServiceBus.RabbitMQ/watch
 - A pull request created by a maintainer must be merged by *another* maintainer. No "self-merges".
 - The pull request must be made from a branch which is a straight line of commits from `develop`. There must be no merges in the branch history since the commit on `develop`.
   - The branch does not have to be based on the latest commit in `develop` but this is preferable, where practical.
+
+## RabbitMQ.Client Updates
+
+In order to reduce the burden of backporting patches to many versions of the package, when a new version of RabbitMQ.Client is released with breaking changes, we will update to that version in a patch release (hotfix on `master`).
```

Comment:
Hmmm... thinking this through a little more, I can't envisage any scenario where we'd have to make a minor bump in order to upgrade the client. I.e. we'd never *add a feature* in order to cater for a breaking change in the client.

I.e. if the client breaking change is not exposed via our API, then we'll make the necessary internal changes to call the new API in the client and then bump the patch version. If the breaking change *is* exposed via our API, we'd have to bump the major version.